Dockerfile

```diff
@@ -1,31 +1,38 @@
-FROM debian:stable-slim AS mirrorbits
+FROM golang:1.23.0 AS build
 
 ## (DL3008)Ignore lint error about apt pinned packages, as we always want the latest version of these tools
 ## and the risk of a breaking behavior is evaluated as low
 # hadolint ignore=DL3008
-RUN apt-get update && \
-  apt-get install --no-install-recommends -y tar curl ca-certificates && \
+RUN apt-get -qq update && \
+  apt-get install --no-install-recommends -y libgeoip-dev tar curl ca-certificates git unzip zlib1g-dev && \
   apt-get clean && \
   rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
-ARG mirrorbits_version=v0.5.1
-RUN mkdir /mirrorbits && \
-  curl -L https://github.com/etix/mirrorbits/releases/download/${mirrorbits_version}/mirrorbits-${mirrorbits_version}.tar.gz -O && \
-  tar xvzf /mirrorbits-${mirrorbits_version}.tar.gz -C /
+# 955a8b2e1aacea1cae06396a64afbb531ceb36d4 introduced go modules (for Go >= 1.11) and commited the generated code from protoc
+# v0.5.1 tag is a bit older (but without these 2 majors elements): https://github.com/etix/mirrorbits/commit/e83e56ac6496a3643d18a731324bd266f75f7b32 commit
+# Diff: https://github.com/etix/mirrorbits/compare/e83e56ac6496a3643d18a731324bd266f75f7b32..955a8b2e1aacea1cae06396a64afbb531ceb36d4
+ARG mirrorbits_version=955a8b2e1aacea1cae06396a64afbb531ceb36d4
```

Comment:
this is very old (2019) I don't think we want this

Reply:
The goal is to stay as close as the current production version as possible. 0.5.1 was released in November 2018, this new commit is January 2019 old.

Looks like the Debian package is also using 0.5.1 (so same code modulo their package fixes)

Reply:
I see, I didn't realise it hadn't been released in so long

Reply:
@timja We are eager to use a more recent version, but there were no released since 2018. We are waiting for https://github.com/etix/mirrorbits/issues/179 of course